src/main/java/br/com/alura/comex/processadores/TipoDeProcessador.java

```diff
@@ -0,0 +1,18 @@
+package br.com.alura.comex.processadores;
+
+import java.util.List;
+
+import br.com.alura.comex.Pedido;
+
+public class TipoDeProcessador {
+
+	public static List<Pedido> getTipoDeProcessador(String opt, String nomeArquivo, List<Pedido> pedidos)
+			throws Exception {
```

Comment:
Esse parâmetro `List<Pedido> pedidos` não está sendo usado. Pode removê-lo!

```suggestion
	public static List<Pedido> getTipoDeProcessador(String opt, String nomeArquivo)
			throws Exception {
```